File: app/src/main/java/com/myself/todo/Database/DadosOpenHelper.java
package com.myself.todo.Database;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;

public class DadosOpenHelper extends SQLiteOpenHelper {
    private static int version = 1;
    private static String name = "agenda.db";
    protected SQLiteDatabase database;
    public DadosOpenHelper(Context context ) {
        super(context, name,null, version);
    }

    @Override
    public void onCreate(SQLiteDatabase sqLiteDatabase) {
        sqLiteDatabase.execSQL( ScriptDLL.getCreateTableCliente() );
        sqLiteDatabase.execSQL( ScriptDLL.getCreateTableAgenda() );

    }

    @Override
    public void onUpgrade(SQLiteDatabase sqLiteDatabase, int i, int i1) {

    }
}

File: app/src/main/java/com/myself/todo/Database/DadosOpenHelper.kt
package com.myself.todo.Database

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper
import org.junit.runner.RunWith

class DadosOpenHelper(context: Context?) : SQLiteOpenHelper(context, name, null, version) {
    protected var database: SQLiteDatabase? = null
    override fun onCreate(sqLiteDatabase: SQLiteDatabase?) {
        sqLiteDatabase.execSQL(ScriptDLL.getCreateTableCliente())
        sqLiteDatabase.execSQL(ScriptDLL.getCreateTableAgenda())
        sqLiteDatabase.execSQL(ScriptDLL.getCreateTableFotos())
        sqLiteDatabase.execSQL(ScriptDLL.getCreateTableMusicas())
    }

    override fun onUpgrade(sqLiteDatabase: SQLiteDatabase?, i: Int, i1: Int) {}

    companion object {
        private const val version = 1
        private val name: String? = "agenda.db"
    }
}